packages/matchbox/src/components/Inline/styles.js

```diff
@@ -0,0 +1,28 @@
+import { system } from 'styled-system';
+
+export const negativeTop = system({
+  gutter: {
+    property: 'marginTop',
+    scale: 'space',
+    transform: (value, scale) => `calc(-${scale[value] || value} - 1px)`
+  }
+});
+
+export const negativeLeft = system({
+  gutter: {
+    property: 'marginLeft',
+    scale: 'space',
+    transform: (value, scale) => `-${scale[value] || value}`
+  }
+});
+
+export const alignChildren = system({
+  align: {
+    property: 'justifyContent',
+    defaultScale: {
```

Comment:
This essentially creates a custom theme scale to map the prop to the correct values